arm-customer-insights/2016-01-01/swagger/customer-insights.json

```diff
@@ -0,0 +1,4735 @@
+            "description": "The name of the hub."
+          },
+          {
+            "name": "interactionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the interaction."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InteractionResourceFormat"
+            },
+            "description": "Parameters supplied to the CreateOrUpdate Interaction operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/InteractionResourceFormat"
+            }
+          },
+          "202": {
+            "description": ""
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "get": {
+        "tags": [
+          "Interactions"
+        ],
+        "operationId": "Interactions_Get",
+        "x-ms-examples": { 
+          "Interactions_Get": { "$ref": "../examples/InteractionsGet.json" } 
+        }, 
+        "description": "Gets information about the specified interaction.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the hub."
+          },
+          {
+            "name": "interactionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the interaction."
+          },
+          {
+            "name": "locale-code",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Locale of interaction to retrieve, default is en-us."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/InteractionResourceFormat"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}/interactions": {
+      "get": {
+        "tags": [
+          "Interactions"
+        ],
+        "operationId": "Interactions_ListByHub",
```

Comment:
Why isn't his modeled as `x-ms-pageable` operation? The result model `"InteractionListResult"` seems to have `nextLink`.